src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsControllerTest.java

```diff
@@ -385,27 +371,51 @@ public void shouldNotSendPrepareForHearingOrderSentEmailWhenNotAgreed() throws E
     }
 
     @Test
-    public void shouldSendContestedApplicationIssuedEmailWhenAgreed() throws Exception {
+    public void shouldSendContestedApplicationIssuedEmailWhenAgreed_andNotifyRespondentSolicitorWhenShould() throws Exception {
+        when(featureToggleService.isRespondentSolicitorEmailNotificationEnabled()).thenReturn(true);
+        when(notificationService.shouldEmailRespondentSolicitor(any())).thenReturn(true);
+
+        buildCcdRequest(CONTESTED_SOL_SUBSCRIBED_FOR_EMAILS_JSON);
+        mockMvc.perform(post(CONTESTED_APPLICATION_ISSUED_CALLBACK_URL)
```

Comment:
have a look at my PR, we should be calling the method directly for unit tests rather than over HTTP